README.md

```diff
@@ -42,15 +42,18 @@ This is used for noticing that a new fee has been applied.
 
 ## Deployment
 
-1. Create a `.env` file using [.env.example](.env.example) and set:
+1. Create a `configs.json` file using [configs.json.example](configs.json.example) and set:
 - `INFURA_PROJECT_ID` your Infura project id
 - `DEPLOYMENT_ACCOUNT_PK` your deploying address private key
 - `GAS_PRICE` gas price to be used, denominated in wei
 - `KYBER_DAO_ADDRESS` [KyberDao address](https://github.com/KyberNetwork/developer-portal/blob/stakingSection/testnet.md#kyberdao)
-- `KYBER_FEE_HANDLER_ADDRESS` [KyberFeeHandler address](https://github.com/KyberNetwork/developer-portal/blob/stakingSection/testnet.md#kyberfeehandler)
+- `KYBER_FEE_HANDLERS_ADDRESS` [`[KyberFeeHandlers addresses]`](https://github.com/KyberNetwork/developer-portal/blob/stakingSection/testnet.md#kyberfeehandler)
```

Comment:
Let's fix this broken link. I think the correct address is https://github.com/KyberNetwork/developer-portal/blob/stakingSection/changelog.md